app-backend/common/src/main/scala/S3.scala

```diff
@@ -0,0 +1,37 @@
+package com.azavea.rf.common
+
+import scala.util.{Success, Failure, Try}
+
+import org.apache.commons.io.IOUtils
+
+import com.amazonaws.auth._
+import com.amazonaws.services.s3.{AmazonS3Client, AmazonS3URI}
+import com.amazonaws.services.s3.model.{S3Object, ObjectMetadata}
+
+import java.io._
+import java.net._
+
+package object S3 {
```

Comment:
@jisantuc I tend to agree but we also only have one example of each thing right now, i.e. Rollbar, `authentication`, and S3, so if we organize the projects by functionality there may be a profusion of mini-projects for a while until we build up some common functionality and combine some of them. Using `common` at the start would take us the other way, we'd have one `common` project that might eventually end up being a bit of a mess, which we would then have to split out into a few smaller common projects.

Neither of those stands out as being clearly superior to me (or clearly horrible) so if anyone has strong opinions one way or the other I'm happy to go along with them.